test/bigtest/tests/units/settings/MappingProfilesForm/TransformationsModal-test.js

```diff
@@ -148,7 +156,35 @@ describe('MappingProfilesTransformationsModal', () => {
         });
 
         it('should display the search pane', () => {
-          expect(modal.searchPane.isPresent).to.be.true;
+          expect(modal.searchPane.isVisible).to.be.true;
+        });
+      });
+    });
+
+    describe('turning off certain record type filters', () => {
+      beforeEach(async () => {
+        await modal.recordTypeFilters(0).clickAndBlur();
+        await modal.recordTypeFilters(1).clickAndBlur();
+      });
+
+      it('should display correct amount of transformation fields', () => {
+        expect(modal.transformations.list.rowCount).to.equal(1);
+        expect(modal.resultsPane.$('[data-test-pane-header-sub]').innerText).to.equal('1 field found');
```

Comment:
I see that we use things like '[data-test-pane-header-sub]' to have access to some internals. I know why we have to use it. But I wonder if we should try to add at least some methods/fields to our interactors or create some util functions. I mean, we can try to locate these dependencies (uncontrolled for us) in one place. @VictorSoroka1  What do you think?

Reply:
Sounds reasonable. I will try the approach with creating the interactor for pane header.

Reply:
I created interactor, I have not extended the one from stripes-components as we don't need other props than the ones which I added. I also added a comment to remove our interactor in future and move the added fields to a shared interactor.